<launch>
  <!--node pkg="pointcloud_segmentation" type="pointcloud_segmentation" name="pointcloud_segmentation" output="screen" respawn="false" launch-prefix="xterm -e gdb -args"-->
  
  <node pkg="pointcloud_segmentation" type="pointcloud_segmentation" name="pointcloud_segmentation" output="screen" respawn="false">
   
    <!-- Topic to subscribe to receive point clouds --> 
    <param name="subscribe_pointcloud_topic" value="/cloud_pcd"/>
    
    <param name="model_type" value="pcl::SACMODEL_PLANE"/>
    
    <param name="method_type" value="pcl::SAC_RANSAC"/>
    
    <param name="set_axis" value="true"/>
    
 
 </node>

</launch>
